README.md

```diff
@@ -1,6 +1,6 @@
 # About
 
-This is early work in progress towards tools and an implementation of the Fix language for Metafacture as an alternative to configuring data transformations with [Metamorph](https://github.com/metafacture/metafacture-core/wiki#morph). The basic idea is to map constructs from Catmandu::Fix like [functions](https://github.com/LibreCat/Catmandu/wiki/Functions), [selectors](https://github.com/LibreCat/Catmandu/wiki/Selectors) and [binds](https://github.com/LibreCat/Catmandu/wiki/Binds) to equivalent constructs from Metafacture like Metamorph [functions](https://github.com/metafacture/metafacture-core/wiki/Metamorph-functions) and collectors ([wiki](https://github.com/metafacture/metafacture-core/wiki/Metamorph-collectors), [commit](https://github.com/metafacture/metafacture-core/commit/0530d6ad72ced992b479bff94d6f56bbef77bb2d)).
+This is early work in progress towards tools and an implementation of the Fix language for Metafacture as an alternative to configuring data transformations with [Metamorph](https://github.com/metafacture/metafacture-core/wiki#morph). Inspired by [Catmandu::FIX](https://github.com/LibreCat/Catmandu/) Metafix processes metadata not as a data stream but as a whole records. The basic idea is to rebuild constructs from Catmandu::Fix like [functions](https://github.com/LibreCat/Catmandu/wiki/Functions), [selectors](https://github.com/LibreCat/Catmandu/wiki/Selectors) and [binds](https://github.com/LibreCat/Catmandu/wiki/Binds) in Java and add additional functionalities that from the Metamorph toolbox.
```

Comment:
"as a whole records" -> "as whole records", "functionalities that from the" -> "functionalities from the"